service_container/definitions.rst

```diff
@@ -92,10 +92,10 @@ fetched from the container::
     
     // adds a new argument with the name of the argument
     // $argumentName = the name of the argument in the constructor
-    $argument = $definition->setArgument('$argumentName', $argumentValue);
+    $definition = $definition->setArgument('$argumentName', $argumentValue);
```

Comment:
```suggestion
    $definition = $definition->setArgument($argumentName, $argumentValue);
```
?

Reply:
That's okay. It's to bind a variable on the definition

Reply:
However, I don't think we need to keep the return. It's confusing between `$definition` & `$argument` which shouldn't be the same "value" for the example.
And we should define an example that can be a real value for `$argumentValue` (new reference or parameter)